### DISCUSSION TITLE
Can we Make LBB copy entire SD Card to a Folder in TARGET/NVME. ?

### DISCUSSION THREAD

**Discussion by keeratsawhney** [outdoorbits/little-backup-box#338] [Ideas]
I am trying to build LBB for personal use. While exploring features, I could not find any option to copy entire sd card to the storage for backup of cards on the go. Can anyone help me with this feature. Would Appreciate that. Also, I believe this would be the fastest way to make a backup of card with this mode.

**Comment by keeratsawhney** [comment]
Also, I noticed When ever I Plug my SD card, LBB shows Time: ?, Computing. I have set the field "" Blank in order to copy all data. But It Seems, It copied the Images in DCIM, for Not Copied Video Files Located in M4ROOT, CLIP as in Sony Cameras SD Card. Please Help

**Reply by outdoorbits** [reply]
Could you please post the log file (just click on Download below the log monitor).

**Comment by keeratsawhney** [comment]
![Screenshot 2024-12-07 at 3 29 24 AM](https://github.com/user-attachments/assets/166f4875-adb9-4a67-a01a-b3515a7e8f55)

**Reply by outdoorbits** [reply]
Which field did you set blank?

**Reply by outdoorbits** [reply]
Do you connect your SD card by cardreader or do you connect your camera?

**Reply by keeratsawhney** [reply]
> Which field did you set blank?

"CAMERA:PATH" The Backup Description Field. Also there seems to be issue in newest update. As soon as I turn on the Box, I get a welcome screen and it shuts off automatically. Not able to get it on. I updated it today.

**Reply by keeratsawhney** [reply]
> Do you connect your SD card by cardreader or do you connect your camera?

I am connecting SD card reader. I am a photographer and I use multiple cameras. So I want to make something portable where I wish to backup Entire SD card for a Cameraman on the go so we have a backup at event itself.

**Reply by keeratsawhney** [reply]
[](https://photos.app.goo.gl/1WAPqoYfUEKxW6pV6) This is what is happening

https://photos.app.goo.gl/1WAPqoYfUEKxW6pV6

**Comment by outdoorbits** [comment]
I couldn't see it perfectly, but obviously there is running a default backup USB to USB? And there is at least one device connected? 
This is not a regular shutdown. I would expect this is a power shortage.  Could you try to start with no peripherals connected?

**Reply by keeratsawhney** [reply]
I guess you are right. I changed supply and it worked. I changed the power supply
![Screenshot 2024-12-07 at 4 13 09 PM](https://github.com/user-attachments/assets/fbef8244-8563-4b5e-859c-bb0d0b3d93cc)

**Reply by keeratsawhney** [reply]
Here is the settings I am talking about. I just want to plug in any sd card of any camera and to have a full copy of it in target device, Is it possible ? Please help, Keeping the Hope High. No Filter Command. Just Basic Backup of Entire SD card to target device without any thumbnails, etc

**Reply by outdoorbits** [reply]
This has to work, it is the most basic function. ;-)
The setting above is just relevant if you connect your camera/smartphone directly to the lbb. If you want to backup a mass storage device like a sd card (by card reader) just start a backup from "USB" or "any USB"  to your target, connect target first and follow the instructions. If you get any problems, please post the log.

**Reply by keeratsawhney** [reply]
I guess it worked. UI makes it look complicated. Thanks. Also could you please let me know a proper way to take logs so I can send it to you if required. Also RED LED on my LBB is flashing twice. Just curious to know is it normal ? Sharing Video of The LED.

https://photos.app.goo.gl/K47h9PyHCAD3uC9UA

**Reply by keeratsawhney** [reply]
Also, is there any way I can later connect my LBB to PC to transfer all the backed up files using USB ? or Something. The fastest way. Also If there is any way I can sync the files to NAS drive to access the data locally instead of cloud.

**Reply by outdoorbits** [reply]
The logs you can download using the Download button below the log monitor. 

The LEDs blinking pattern signals the state, for example waiting for a device or backup running. Not too interesting for you as you see everything on the monitor.

To get data from LBB to a local computer, it's best to use the Windows share or ftp server of LBB.

**Reply by outdoorbits** [reply]
Or you can backup your SSD to an USB device...
